db/seeds.rb

```diff
@@ -35,7 +35,7 @@ def create_category(name, kids: nil, parent_id: nil)
 Document.create!(name: "Chicago Tool Library Code of Conduct", code: "code_of_conduct", summary: "Defines acceptable behavior for CTL")
 
 ActsAsTenant.with_tenant(
-  Library.create!(name: "Chicago Tool Library", hostname: "chicago.local.chicagotoollibrary.org")
+  Library.create!(name: "Chicago Tool Library", hostname: "chicago.local.chicagotoollibrary.org", member_postal_code_pattern: "60707|60827|^606")
```

Comment:
Maybe add this to `db/migrate/20200924072906_establish_tenant_zero.rb` also?

Reply:
Updated in 967092d00f.